root.go

```diff
@@ -5,9 +5,19 @@ import (
 	"github.com/suborbital/subo/subo/command"
 	"github.com/suborbital/subo/subo/features"
 	"github.com/suborbital/subo/subo/release"
+	"github.com/suborbital/subo/subo/util"
 )
 
 func rootCommand() *cobra.Command {
+	defer func() {
+		version_msg, err := release.CheckForLatestVersion()
+		if err != nil {
+			util.LogFail(err.Error())
+		} else if version_msg != "" {
+			util.LogInfo(version_msg)
+		}
+	}()
```

Comment:
Let's move this into `main.go` since it's not part of the `cobra.Command` stuff. `defer` works fine here, but it doesn't make it concurrent. Something like this:

```go
func main() {
	rootCmd := rootCommand()

	done := update()
	if err := rootCmd.Execute(); err != nil {
		os.Exit(-1)
	}

	<-done
}

func update() chan bool {
	done := make(chan bool)

	go func() {
		version, err := release.CheckForLatestVersion()
		if err != nil {
			util.LogFail(err.Error())
		} else if version != "" {
			util.LogInfo(version)
		}
		done <- true
	}()

	return done
}
```